app-ios/Modules/Sources/Timetable/InfoChip.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+struct InfoChip: View {
+    
+    private let label: String
+    private let labelColor: Color
+    private let strokeColor: Color?
+    private let backgroundColor: Color
+    
+    public init(
+        label: String,
+        labelColor: Color,
+        strokeColor: Color? = nil,
+        backgroundColor: Color = .clear
+    ) {
+        self.label = label
+        self.labelColor = labelColor
+        self.strokeColor = strokeColor
+        self.backgroundColor = backgroundColor
+    }
+    
+    var body: some View {
+        Text(label)
+            .font(Font.system(size: 12, weight: .medium))
+            .padding(.horizontal, 8)
+            .padding(.vertical, 4)
+            .foregroundColor(labelColor)
+            .background(backgroundColor)
+            .cornerRadius(4)
+            .overlay(
+                RoundedRectangle(cornerRadius: 4)
+                    .inset(by: 0.5)
+                    .stroke(strokeColor != nil ? strokeColor! : .clear, lineWidth: 1)
```

Comment:
It looks better to apply overlay only if `strokeColor` exists, but I didn't know how to do that 😢

Reply:
@ry-itto 
Thank you for your advice!
In fact, I've already tried exactly the same way, but the following error happened.
> Branches have mismatching types 'some View' and 'EmptyView'
![error](https://github.com/DroidKaigi/conference-app-2023/assets/43767445/0ec44ec6-c3b3-4fd5-82f1-fae061c8f7cd)

I thought this code was valid since EmptyView conformed View. So, I was not sure how to resolve it 😢

Reply:
You should use `overlay(alignment:content:)` instead of `overlay(_:alignment:)`.

```swift
. overlay { ... } // <- braces!
```

Reply:
@woxtu 
Oh... I haven't noticed that there were some overlay functions... 😮
Thank you so much for your help!

@ry-itto 
And sorry, you wrote the same code several day ago but I could not find the diff 🙇‍♂️
I think all TODOs have been resolved, so please review it again when you have time!

https://github.com/DroidKaigi/conference-app-2023/pull/724/commits/22185235c6afddd2d61738ebb3ce1d630c1f66b2